webapp/views/App/views/Designer/CategoryList/TableRow/store/state/state.js

```diff
@@ -7,6 +7,7 @@ export const keys = {
   inCategoriesPath: 'inCategoriesPath',
```

Comment:
I'm thinking that this state is initialized by each row. would it make sense to move it a level up (CategoryList state) ?

Reply:
it will be removed from TableRow state, it won't be used anymore